### PR TITLE
array-manipulation-1 - Alex Gonzalez

### PR DESCRIPTION
**Pull request by gonzalez-alex**
array-manipulation-1

### CODE REVIEW

File: array-manipulation-1/src/initial.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - if index does not equal last index
+        - push array elements into new array
+- return new array
+*/
+
+function initial(array) {
+  var newArray = [];
+  var lastIndex = array.length - 1;
+  for (var i = 0; i < array.length; i++) {
+    if (i !== lastIndex) {
```
Comment (codingcodymiller) [inline array-manipulation-1/src/initial.js]: If you stop the loop one index sooner, you won't need this conditional 🤓

File: array-manipulation-1/src/tail.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- loop through the array parameter
+    - if index is greater than 0
+        - add current array element to new array
+- return new array
+*/
+
+function tail(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i >= 1) {
```
Comment (codingcodymiller) [inline array-manipulation-1/src/tail.js]: If you start the loop at `1`, you won't need this conditional 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by gonzalez-alex** [comment]
updated initial.js condition and tail.js initial expression to remove unnecessary conditionals**

is there any way to edit your commit message after already committing? realized my commit message wasn't totally accurate haha.

File: array-manipulation-1/src/tail.js (hunk 2 above)
Comment (gonzalez-alex) [reply array-manipulation-1/src/tail.js]: brain just defaulted the 'for loop' and ended up making this more complicated than it should be.. haha got to stop these bad habits early

**Review by codingcodymiller — APPROVED** [review]

**Comment by codingcodymiller** [comment]
> is there any way to edit your commit message after already committing? realized my commit message wasn't totally accurate haha.

There is! You can run `git commit --amend` after committing, and it will give you an option to revise the message

**Pull request merged**